app/models/queue_item.rb

```diff
@@ -0,0 +1,22 @@
+class QueueItem < ActiveRecord::Base
+  belongs_to :video
+  belongs_to :user
+
+  def video_title
+    self.video.name
```

Comment:
No need to use `self` here. You are using `#video` getter provided by `belongs_to :video`. If you would set the video `#video=` then you would have to use `self`.